news/1673.bugfix

```diff
@@ -0,0 +1 @@
+Fix broken relations info. @ksuess
```

Comment:
@davisagli @ksuess don't we have an issue for this one? We currently use the ID of the Pull Request and not the issue ID if I am not mistaken here. This should be fixed before making a release.